_pages/governance.md

```diff
@@ -255,7 +255,7 @@ working and interest groups.
 
 The current BEP Working Groups and their leads can be found in the
 section on [BIDS Extension
-Proposals](https://bids-specification.readthedocs.io/en/stable/06-extensions.html#bids-extension-proposals){:target="_blank"} 
+Proposals](https://bids.neuroimaging.io/get_involved.html#extending-the-bids-specification){:target="_blank"}
```

Comment:
there is probably a nicer way to use some "local" jekyll link here, because we are linking to a page that is in the source here. Help welcome, else this also works "as is".

Reply:
Let's keep it as is for now and maybe open a "not urgent issue": my jekyll magic is not that good yet